src/main/java/time/TimeInCapitals/showtime/LocalTimeService.java

```diff
@@ -36,4 +47,27 @@ public String getTimeAndDate() {
 		return localTimeAndDate;
 	}
 
+	public String changeTheFirstLetterToUppercase(String word) {
```

Comment:
Coś strasznie złożona ta metoda w odniesieniu do tego, co obiecuje robić. Co oznacza `regexResults` i czemu właściwie sprawdzamy tego długość?

Reply:
regexResults przechowuje w tabeli wyrazy po wykonaniu na nich metody split, czyli podział przy użyciu wyrażenia regularnego, ale po zmianach które chcę dokonać ta metoda już nie będzie potrzebna. Potrzebowałem metodę która zamieniałaby pierwszą literę miasta na dużą, tak żeby mogła ją przyjąć metoda ZoneId.of("Continent/City");

Reply:
Tzn? Jakie zmiany?

Reply:
Tak, obecnie :) Komentarz pisałem jeszcze przed zmianami :)

Reply:
Po tym, jak już ogarnęliśmy kwestie springowe, to możemy wrócić do tego miejsca:

1. Na pewno nie mamy do czynienia z regexem, więc proszę o zmianę nazwy pola na bardziej adekwatną.

Reply:
Poprawione. regexResults zmieniłem na dataFromFile